### PR TITLE
chrome_cast.feature: Added a section explaining some of the terms tha…

### PR DESCRIPTION
**Pull request by attackaljackal**
…t might not have been clear to new users

### CODE REVIEW

File: manual_tests/chrome_cast.feature [hunk 1]
```diff
@@ -5,15 +5,26 @@ Feature: Checking core functionality of Google Chromecast works correctly when c
   I want to check core functionality of Chromecast with SMP
   So I can ensure that it works as expected 
 
+# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+
+  Terms you may need to know
+  - SMP:                     Standard Media Player. The player used by clients (e.g. iplayer, sounds, news). It unifies video playback across the BBC.
+  - allowcasting:            This allows cast/playback
```
Comment (dalyw01) [inline manual_tests/chrome_cast.feature]: On a chromecast perhaps?

Comment (attackaljackal) [reply manual_tests/chrome_cast.feature]: It works on my machine

Comment (dalyw01) [reply manual_tests/chrome_cast.feature]: Understood.

File: manual_tests/chrome_cast.feature [hunk 2]
```diff
@@ -9,8 +9,8 @@ Feature: Checking core functionality of Google Chromecast works correctly when c
 
   Terms you may need to know
   - SMP:                     Standard Media Player. The player used by clients (e.g. iplayer, sounds, news). It unifies video playback across the BBC.
-  - allowcasting:            This allows cast/playback
-  - allowContinlousCast:     This allows multiple casts to be played one after the other
+  - allowcasting:            This allows cast/playback on Chromecast.
```
Comment (dalyw01) [inline manual_tests/chrome_cast.feature]: You are partially correct. Needs to be FROM SMP on to the Chromecast

**Comment by dalyw01** [comment]
Looks good

**Pull request merged**